### DISCUSSION TITLE
Make custom scrollbar thumb show below matches

### DISCUSSION THREAD

**Discussion by DieSpinne** [qutebrowser/qutebrowser#5780] [Q&A] [answered]
I have customized Qutebrowser's scrollbar with a CSS stylesheet. All is OK, except for the `/` search match marks, which are "hidden behind" the scrollbar thumb:

![09-12:02:19a](https://user-images.githubusercontent.com/62444221/95603903-4c9b8a00-0a2d-11eb-8b4b-474c12a1ace8.png)

If I use a semi-transparent color for the thumb, the marks are visible, but it is still difficult to see them.

![09-12:39:43a](https://user-images.githubusercontent.com/62444221/95603962-63da7780-0a2d-11eb-8615-747ece939ea1.png)

Any way/idea to make the marks show above the scrollbar thumb, just as they show above the scrollbar track? If I decrease transparency to a value below 0.5, marks are more visible but the thumb starts to disappear as well.

Here is the relevant part of the stylesheet:
```
::-webkit-scrollbar {
  width:  14px;
  height: 14px;
  background-clip: border-box;
}

::-webkit-scrollbar-thumb {
  background-color: rgba(0,255,255,0.5);
  border: 2px solid black;
}

::-webkit-scrollbar-track {
  background-color: lightgray;
  border: 2px solid black
}
```

Thank you for the help.

**Comment by The-Compiler** [comment, marked as answer]
I don't think this is possible - the scrollbar (as well as the highlighted matches) is coming from Chromium.

**Reply by DieSpinne** [reply]
Alright, thanks for your response. Do notice that I have successfully styled the scrollbar as shown above, although not the matches, even though both come from Chromium.

**Reply by The-Compiler** [reply]
Yep - Chromium lets you style the scrollbar (mostly inheriting that functionality from WebKit, where this [was added in 2009](https://webkit.org/blog/363/styling-scrollbars/)), but not the tickmarks. This isn't specific to qutebrowser though, same story with Google Chrome (and presumably other Chrome-based browsers) as well.
